src/Http/Controllers/Support/ResolveController.php

```diff
@@ -122,4 +123,28 @@ public function resolveIdsToNames(Request $request)
 
         return response()->json($response);
     }
+
+    public function resolveMainCharacter(Request $request)
+    {
+
+        // Init the initial return array
+        $response = collect();
+        // Grab the ids from the request for processing
+        collect(explode(',', $request->ids))->map(function ($id) {
+
+            // Convert them all to integers
+            return (int) $id;
+        })->each(function ($chunk) use (&$response) {
+
+            $character = User::find($chunk);
+            if (is_null($character)) {
```

Comment:
it can be switch to a ternary `$maincharacter = is_null($character) ? null : $character->group->main_character;`